openstf/pipeline/predict_sklearn.py

```diff
@@ -1,18 +1,19 @@
-from datetime import datetime, timedelta
```

Comment:
Just a general remark about naming. The previous convention was to name the pipeline modules after the tasks, `create_forecast`, `train_model`, `optimize_hyperparameters` etc. If we decide that the old convention is not suitable anymore, we should design a new one. I'm not sure if `predict_sklearn` is a good module name. I don't think we should postfix the pipeline with the name of one of the packages that is used (sklearn).

Reply:
I agree we should revert to the previous convention once the new pipeline is more stable.

Reply:
Sounds good. I don't mind if you change it now, but like you said we can also do it later I guess. Probably more important to keep moving with these changes and get the code merged.

Reply:
Yes I agree, I used the postfix _sklearn to distinguish from the old pipeline when both pipelines were still in the branch. I agree with moving back to the old naming convention.